aws-datastore/src/main/java/com/amplifyframework/datastore/syncengine/Orchestrator.java

```diff
@@ -305,9 +322,12 @@ private Completable startApiSync() {
 
     private void stopApiSyncBlocking() {
         try {
-            stopApiSync()
+            boolean stopped = stopApiSync()
                 .subscribeOn(startStopScheduler)
-                .blockingAwait(OP_TIMEOUT_SECONDS, TimeUnit.SECONDS);
+                .blockingAwait(LOCAL_OP_TIMEOUT_SECONDS, TimeUnit.SECONDS);
```

Comment:
This is probably actually be a _network_ op, since it has to send the `stop` messages on the WebSocket subscription. May want to use the long timeout for this one.